homeassistant/components/ambient_network/const.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for the Ambient Weather Network integration."""
+
+from datetime import timedelta
+import logging
+
+DOMAIN = "ambient_network"
+
+API_LAST_DATA = "lastData"
+API_STATION_COORDS = "coords"
+API_STATION_INDOOR = "indoor"
+API_STATION_INFO = "info"
+API_STATION_LOCATION = "location"
+API_STATION_NAME = "name"
+API_STATION_MAC_ADDRESS = "macAddress"
+API_STATION_TYPE = "stationtype"
+
+LOGGER = logging.getLogger(__package__)
+
+SCAN_INTERVAL = timedelta(minutes=5)
```

Comment:
Remove it from here, it's only used in coordinator.py
```suggestion
```

Reply:
Fixed.